index.html

```diff
@@ -0,0 +1,206 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Alina Sredna Week 1 Project</title>
+    <link href='https://fonts.googleapis.com/css?family=Oswald:400,300|Lato:400,100|Merriweather|Lora:400,700italic,400italic' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <!-- top of page -->
+     <section class="purple-rectangle"></section>
+     <header class="volume">VOLUME 3-SEPTEMBER 2013</header>
+
+    <!-- CSS OFF -->
+    <header class="cssoff">CSS<br>OFF</header>
+
+    <!-- barbie -->
+    <section class="barbie">
+      <figure>
+        <img class="barbie-pic" src="img/barbie.png" alt="Barbie" title="Barbie">
+      </figure>
+      <article class="barbietext">
+        <header class="box-model-header">
+          <h3><strong><em>Box Model</em></strong></h3>
```

Comment:
I would argue that you are semantically exploding here. The `h3` already gives it precedence in the outline, while you are also saying that there is emphases on this string and not to mention that it's `strong`?! 

Also, when nesting elements, I would suggest actually visually nesting the elements. '

```html
<h3>
  <strong>
    <em>
      hello world!
```

This just makes it easier to read the code.